docs/topics/mtr-rn-resolved-issues-1-2-0.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * docs/release-notes-mtr/mtr_release_notes-1.0/master.adoc
+
+:_content-type: REFERENCE
+[id="mtr-rn-resolved-issues-1-2-0_{context}"]
+= Resolved issues
+
+For a complete list of all issues resolved in this release, see the list of link:https://issues.redhat.com/browse/WINDUP-3632?filter=12419395[{ProductShortName} 1.2.0 resolved issues] in Jira.
```

Comment:
@anarnold97 @HagayVider1 
This link is wrong.

Reply:
Fixed - https://issues.redhat.com/browse/WINDUP-3632?filter=12419395

Reply:
@anarnold97 
But this seems to be the same link?